subprojects/docs/src/docs/userguide/build_cache.adoc

```diff
@@ -43,31 +43,31 @@ Put `org.gradle.caching=true` in your `gradle.properties`::
 Gradle will try to reuse outputs from previous builds for all builds, unless explicitly disabled with `--no-build-cache`.
 
 When the build cache is enabled, it will store build outputs in the Gradle user home.
-For configuring this directory or different kinds of build caches see <<sec:build_cache_configure>>.
+For configuring this directory or different kinds of build caches see <<#sec:build_cache_configure,Configure the Build Cache>>.
 
 [[sec:task_output_caching]]
-=== Task Output Caching
+== Task Output Caching
 
-Beyond incremental builds described in <<sec:up_to_date_checks>>, Gradle can save time by reusing outputs from previous executions of a task by matching inputs to the task.
+Beyond incremental builds described in <<more_about_tasks.adoc#sec:up_to_date_checks,up-to-date checks>>, Gradle can save time by reusing outputs from previous executions of a task by matching inputs to the task.
```

Comment:
The section is called "Up-to-date checks (AKA Incremental Build)". Is it intentional not to put the full section name in here?

Reply:
it is, but if you prefer adding the "incremental build" bit I can.

Reply:
That's fine.